### ISSUE TITLE
Doesn't work with iOS swipe

### ISSUE THREAD

**Issue by ghost** [marcglasberg/back_button_interceptor#15]
The package description makes no mention of handling the iOS back swipe. It would be appreciated to update the package description to explicitly state that the iOS swipe is not supported (I had to download and test the sample in the iOS simulator to confirm). It would also be helpful if the description described whether iOS support is planned for a future release.

**Comment by marcglasberg** [comment]
Yes, the Android back-button and the iOS swipe are very different features, and are implemented in very different ways in the Flutter framework. This package is very focused on the back-button and will not support iOS swipe. I will add this info to the documentation. Thanks!

**Comment by Abdelaty** [comment]
if you want to detect swipe actions you can check this https://stackoverflow.com/questions/55050463/how-to-detect-swipe-in-flutter
